### DISCUSSION TITLE
Pre-push hook

### DISCUSSION THREAD

**Discussion by svalexander** [trussworks/react-uswds#1910] [Q&A]
Hi, I've forked the repo in order to push up my local branch for issue #1619 and then create a pr from it. When I try to push it fails b/c there's an error when the pre-push hook is run. Running "yarn danger local" in the CLI I can see the issue better - localGetDiff throws an error. The error is as follows: Could not get diff from git between master and HEAD.
From the contributing documentation I see that any changes to Storybook are automatically checked in Happo.io. Am I correct in assuming b/c I've added a story the pre-push hook is failing as my local diff doesn't match what's in Happo? And how do I proceed? Is it best to bypass the hook?

Just going to tag a few contributors randomly on this, not sure who checks the repo regularly for discussions. @ahobson @brandonlenz @haworku @tinyels

**Comment by brandonlenz** [comment]
@svalexander Hi! I'm sorry, I didn't see this discussion at all (I think you're the first to use Discussions for this repo!).

I see that you figured it out and created https://github.com/trussworks/react-uswds/pull/1911. Did you figure out what the issue you were facing was? In case it happens to others in the future, I'd love for us to be able to be aware of it and potentially do something to address it.

**Reply by werdnanoslen** [reply]
@svalexander were you able to resolve this? We've had many Storybook PRs since this discussion from the community, but perhaps we can change the documentation in a way that covers what you encountered.
